docs/css/avatar.css

```diff
@@ -0,0 +1,42 @@
+.bat-avatar-xl {
```

Comment:
you should make one basic avatar and then you should make its variation then it will be less code and you will not have to repeat the border -radius and all the basic style in every class too.

Reply:
Done